src/pages/climbs/[id].tsx

```diff
@@ -378,14 +385,18 @@ export const getStaticProps: GetStaticProps<ClimbPageProps, { id: string }> = as
     }
   }
 
-  const sortedClimbsInArea = await fetchSortedClimbsInArea(climb.ancestors[climb.ancestors.length - 1])
+  const parentAreaData = await getArea(climb.ancestors[climb.ancestors.length - 1])
```

Comment:
Good that you reuse getArea().  By default the call hits the API. Since we're not editing here and can afford cached data, can you add a 2nd param (for fetchPolicy), `cache-first`?